content/general/concepts/boolean/boolean.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'Boolean'
+Description: 'The boolean data type represents the logical values of true and false.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Data Types'
+  - 'Types'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Booleans hold a true or false value and are mostly used in conditional statements to control a program's flow of execution. This data type is named after George Boole, a 19th century English logician.
+
+Some languages use comparison operators (`>=`, `==`, `is`, `not`, etc.) to produce boolean values, while others may use "falsy" values like `""`, `null`, or `0`. Some even use a combination of both of these approaches.
+
+Because functions are derived from objects, their closest classification is with composite data types.
```

Comment:
@ethan-master-coding Was this supposed to be with `function.md`?

Reply:
@Dusch4593 I think only line 19. Let me move that now